source/Grid/Grid.js

```diff
@@ -63,6 +64,12 @@ export default class Grid extends Component {
     noContentRenderer: PropTypes.func.isRequired,
 
     /**
+     * Optional callback invoked on mouse-wheel activity.
+     * (event: SyntheticEvent): void
+     */
+    onWheel: PropTypes.func,
```

Comment:
so I changed it to just accept an onWheel prop but not default it to anything, so you aren't getting any de-opt and I think its reasonable for users to want to listen to that event in general (I have for other reasons)

If you think its a bad idea though to allow listening for wheel events at all i can remove it.